lib/mavenlink/model.rb

```diff
@@ -13,6 +13,12 @@ def self.collection_name
       (name || "undefined").split(/\W+/).last.tableize.pluralize
     end
 
+    # @param models [Array<Hash>]
+    # @return [Array<Mavenlink::Model>]
+    def self.bulk_create(models)
+      scoped.bulk_create(models)&.results
```

Comment:
What is the reason for the safe-traversal operator (`&`)? Can `bulk_create` return `nil`? What does that mean?

Reply:
@brandonduff that is an artifact of a few things we were trying. `bulk_create` will always return a `Mavenlink::Response` and never `nil`, so it should be safe to remove it.